data_steward/cdr_cleaner/cleaning_rules/remove_extra_tables.py

```diff
@@ -123,11 +55,22 @@ def __init__(self,
         this SQL, append them to the list of Jira Issues.
         DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
         """
-        desc = 'Remove any tables that are not OMOP, OMOP extension, or Vocabulary tables.'
+        desc = 'Remove any tables that are not OMOP, OMOP extension, Vocabulary or AOU custom tables.'
+        # Use custom cdr_metadata instead of metadata
+        cdm_achilles_vocab_tables = list(
+            set(
+                cdm_schemas(include_achilles=True, include_vocabulary=True).
+                keys()) - {'metadata'}) + ['_cdr_metadata']
```

Comment:
you would need a metadata table as it is one of the tables added in CDM 5.3 https://ohdsi.github.io/CommonDataModel/cdm53.html#Metadata_Tables

Reply:
I thought we were using `_cdr_metadata` instead of that. Are we using both?